docs/index.md

```diff
@@ -22,6 +22,7 @@ Supercharge your development workflow.
 * [auth0 actions](auth0_actions.md)	 - Manage resources for actions
 * [auth0 apis](auth0_apis.md)	 - Manage resources for APIs
 * [auth0 apps](auth0_apps.md)	 - Manage resources for applications
+* [auth0 attack-protection](auth0_attack_protection.md)	 - Manage attack protection settings
```

Comment:
Leaving things a bit better than how we found them, attack protection link was missing in the docs.